game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyLogin.java

```diff
@@ -57,8 +57,9 @@ public void promptLogin() {
               .title("Lobby Not Available")
               .dialogText(
                   String.format(
-                      "Your version of TripleA is too old, please download the latest: <a>%s</a>",
-                      UrlConstants.DOWNLOAD_WEBSITE))
+                      "Your version of TripleA is too old, please download the latest:"
+                          + "<br><a href=%s>%s</a>",
```

Comment:
This might work, because HTML parser are not known for their strictness, but usually the URL is put in quotes

Reply:
Probably better to not leave it to *might*, agree with the call-out, updated: 1e9afeb45